rpc_jobs/rpc_openstack.yml

```diff
@@ -139,6 +139,31 @@
     jobs:
       - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
 
+# TODO(mattt): We need to split out existing pre-merge jobs between check and
+#              gate, but this will be done after rpc-openstack has been
+#              successfully added to the release pipeline.
+
+- project:
+    name: 'rpc-openstack-mnaio-release'
+    repo_name: 'rpc-openstack'
+    repo_url: 'https://github.com/rcbops/rpc-openstack'
+    branch:
+      - pike
+      - newton
```

Comment:
The 'newton' branch cannot do a 'no_artifacts' job. It only has the 'loose_artifacts' job option. This will need to be split into two different project definitions, or use exclusions.